cypress/support/commands.ts

```diff
@@ -53,6 +53,20 @@ Cypress.Commands.add("logout", () => {
   });
 });
 
+Cypress.Commands.add("seedQueries", (queries) => {
```

Comment:
Can you make the default value here to set the queries you have above? I think that will be easier for folks who just want some queries in the tests but don't really care what they are.

Reply:
I agree since we will be using these queries won't change testing user to user, might as well add them as the default value